src/mcts/search.cc

```diff
@@ -368,9 +368,11 @@ std::vector<std::string> Search::GetVerboseStats(Node* node) const {
   std::vector<std::string> infos;
   for (const auto& edge : edges) {
     float Q = edge.GetQ(fpu, draw_score, logit_q);
+    const auto parent_m = node->GetM();
+    const auto child_m = edge.GetM(parent_m);
```

Comment:
Just in case anyone else is wondering - yes this is wrong, should be parent_m+1, but the same 'error' exists in the pick node code.
Same applies on the line below - child_m - parent_m should really be child_m - parent_m - 1 - but same problem exists in pick node.

Reply:
I believe you meant the child move default m has an off-by-1  error, but it should be 1 move fewer (not `parent_m+1`). Filed #1324